tests/test_unit_system.py

```diff
@@ -26,8 +23,13 @@
 from tests.utils import array_equality_override, to_file_from_file_test
 
 
+@pytest.fixture
+def setup_array_equality_override():
+    return array_equality_override
+
+
 class DataWithUnits(pd.BaseModel):
-    l: LengthType = pd.Field()
+    L: LengthType = pd.Field()
```

Comment:
Ruff does not like single l character as variable name as this is said to be easily confused with other letters (like 1 or i) when using certain fonts.

Make sense to me so I changed it. Is their any backend data structures that also uses this. I do not want to break consistency.

Reply:
this is ok